client/components/notice/style.scss

```diff
@@ -284,4 +284,8 @@ a.notice__action {
 			opacity: 1;
 		}
 	}
+
+	& + & {
+		margin-top: 8px;
+	}
```

Comment:
This rule is for `.notice.is-compact` to add some spacing between multiple consecutive compact notices.